Modules/Core/Common/include/itkVariableLengthVector.h

```diff
@@ -1052,7 +1052,7 @@ struct GetType
  * \sa \c VariableLengthVectorExpression
  */
 template <typename TExpr1, typename TExpr2>
-inline typename mpl::EnableIf<mpl::And<mpl::IsArray<TExpr1>, mpl::IsArray<TExpr2>>, unsigned int>::Type
+inline typename std::enable_if<mpl::And<mpl::IsArray<TExpr1>, mpl::IsArray<TExpr2>>::Value, unsigned int>::type
```

Comment:
@dzenanz Nice! however, with C++14 `std::enable_if_t`, it can be even nicer 😸  No need for `typename` and `::type` anymore:

    inline std::enable_if_t<mpl::And<mpl::IsArray<TExpr1>, mpl::IsArray<TExpr2>>::Value, unsigned int>

Reply:
You are right! Thank you for early feedback.